lib/locales/urhobo/names/female_first_name.js

```diff
@@ -0,0 +1,34 @@
+module.exports = [
+  'Oghenetega',
+  'Oghenevwede',
+  'Miriodere',
+  'Omonigho',
+  'Akpenvwoghene',
+  'Oghenekevwe',
+  'Oghenenyerhovwo',
+  'Oghenerhoro',
+  'Ogheneruemu',
+  'Oghenerukevwe',
+  'Oghenetega',
+  'Ogheneruno',
+  'Asaroyoma',
+  'Obatare',
+  'Oborerhinrin',
+  'Okiremute',
+  'Onoriode',
+  'Ufuoma ',
+  'Uruemuesiri ',
+  'Emetemedia ',
+  'Enajemete',
+  'Omotejohwo',
+  'Omotore ',
+  'Umukoko ',
+  'Umuto',
+  'Onanefe ',
+  'Omotore ',
+  'Efediome',
+  'Efevwerha',
+  'Oghonoro',
+  'Oboganriemu',
+];
+  
```

Comment:
You might want to lose the trailing spaces on this line, that's what's causing this red warning here.

The empty line is important but the extra spaces on the line isn't.